console/bridge.go

```diff
@@ -432,6 +432,8 @@ func (b *bridge) Send(call jsre.Call) (goja.Value, error) {
 			}
 		case rpc.Error:
 			setError(resp, err.ErrorCode(), err.Error())
+		case rpc.DataError:
+			resp.Set("error", map[string]interface{}{"code": -32603, "message": err.Error(), "data": err.ErrorData()})
```

Comment:
I think it would be better to not have this as a separate case here. Instead, check for the `rpc.DataError` case in the `rpc.Error` case and assign the "data" if the type assertion succeeds.